platform/core/src/utils/addServer.test.js

```diff
@@ -6,8 +6,9 @@ describe('addServers', () => {
       {
         name: 'DCM4CHEE',
         wadoUriRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/wado',
-        qidoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
-        wadoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
+        qidoUrlPrefix: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
```

Comment:
Should all be replaced with just uri.  Then, if necessary, specific versions can include the prefix definitions as needed.